terraform-post-packer/backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    encrypt        = true
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
+    key            = "skeleton-packer/terraform-post-packer.tfstate"
```

Comment:
This key needs to be updated.

Reply:
Fixed in https://github.com/cisagov/nessus-packer/pull/3/commits/81625b9ef2298d0f606ae87cb100f5a4109cfcaf.